skupper/skupper/galaxy.yml

```diff
@@ -8,7 +8,7 @@ namespace: skupper
 name: skupper
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.6.0
+version: 1.8.0
```

Comment:
Can you set it to 1.8.3 and also update the following value ?
https://github.com/skupperproject/skupper-ansible/blob/4cd639ddb502c65d1055a28c75d8a1da7df9732f/skupper/skupper/roles/skupper_cli_install/defaults/main.yml#L3